### PR TITLE
Update bitcoin address settings

### PR DESCRIPTION
**Pull request by alex-miller-0**
Adds functionality for native Bitcoin segwit addresses and transactions. This is done by adding a settings option to switch between address types, which include legacy, wrapped segwit, and native segwit.

**NOTE:** This can be merged now because I commented out the segwit option, which is not yet supported by firmware.

### CODE REVIEW

**Comment by alex-miller-0** [comment]
**DO NOT MERGE until the SDK can be updated in package.json properly (i.e. after SDK release)**

**Comment by alex-miller-0** [comment]
I have disabled the segwit option for now so we can release this before segwit is enabled in firmware

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "elliptic": "^6.5.4",
     "ethers": "^5.0.17",
     "gridplus-react-crypto": "0.0.5",
-    "gridplus-sdk": "^0.9.0",
+    "gridplus-sdk": "GridPlus/gridplus-sdk#add-bip84",
```
Comment (alex-miller-0) [inline package.json:17]: this needs to be undone before merge

**Review by KevinVitale — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (alex-miller-0) [reply package.json]: 02960ca

**Pull request merged**